helm-charts/jitsi-conference-mapper/templates/hpa.yaml

```diff
@@ -0,0 +1,47 @@
+{{- /*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/}}
+
+{{- if .Values.autoscaling.enabled }}
```

Comment:
this is part of the (auto generated) helm template ($helm create <name>) I also see the responsibility at the devOps Team here (?)

Reply:
You decide these things, since you provide a IaaS solution for our customer. Do you think, that the service has to scale (e.g. high load)? If yes, we would need to provide a mechanism to work with different nodes (e.g. db). If no, please remove unnecessary settings and if clauses

Reply:
(I guess the template of the helm chart you were using to adapt it for this app is providing a mutli-node setup.)